compiler/rustc_middle/src/ty/context.rs

```diff
@@ -961,6 +961,7 @@ pub struct FreeRegionInfo {
 /// [rustc dev guide]: https://rustc-dev-guide.rust-lang.org/ty.html
 #[derive(Copy, Clone)]
 #[rustc_diagnostic_item = "TyCtxt"]
+#[cfg_attr(not(bootstrap), rustc_pass_by_value)]
```

Comment:
This is my first time changing something on the compiler, which makes me have to use this cfg. Do we merge the PR with this in place or is there something I should change here before we merge? When do we clean this up?

Reply:
yes that's fine, these `cfg_attr` get removed the next time we bump the bootstrap compiler, e.g. in https://github.com/rust-lang/rust/pull/90042/files